### PR TITLE
Fixes walls under doors breaking to space

### PR DESCRIPTION
**Pull request by DaedalusGame**
:cl:
fix: fixed walls under doors breaking to space
tweak: changed doors to no longer spawn on top of walls
/:cl:

### CODE REVIEW

**Comment by praisenarsie** [comment]
AAAAAAAAAAAAAAAAA

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -133,9 +133,12 @@
 			if(1 to 9)
 				var/turf/here = get_turf(src)
 				for(var/turf/closed/T in range(2, src))
-					here.ChangeTurf(T.type)
-					return INITIALIZE_HINT_QDEL
```
Comment (ChangelingRain) [inline code/game/machinery/doors/airlock.dm]: pretty sure you should keep the hint?

Comment (DaedalusGame) [reply code/game/machinery/doors/airlock.dm]: LateInitialize can't use it.

**Comment by 81Denton** [comment]
Finally!

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**